code/_onclick/telekinesis.dm

```diff
@@ -144,8 +144,8 @@ var/const/tk_maxrange = 15
 		apply_focus_overlay()
 		focus.throw_at(target, 10, 1,user)
 		last_throw = world.time
-		user.changeNext_move(CLICK_CD_MELEE)
```

Comment:
Or maybe it's not a "nerf" and is there for a reason?

Please link to the /tg/ PR that this reverts if you can find it.